instrumentation/spring/spring-webmvc/spring-webmvc-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webmvc/v5_3/WebMvcTelemetryProducingFilter.java

```diff
@@ -60,21 +60,20 @@ public void doFilterInternal(
     }
 
     Context context = instrumenter.start(parentContext, request);
+    if (httpRouteSupport.hasMappings()) {
```

Comment:
I suspect that it was deliberately added after `filterChain.doFilter`.

Reply:
thanks, reverted changes to these two files